README.md

```diff
@@ -1 +1,45 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse
+
+> A simple - Hello Microverse - HTML and CSS project with linters
+
+
+## Built With
+
+- HTML and CSS
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Have a working operating system either Windows, Mac OS or Linux.
+
+### Setup
+Create the directory in which you'll like to clone this project.
+
+### Install
+Navigate to the created directory above
+Run git clone 'the project's https or ssh'
+
+### Usage
+View the HTML document in your preferred browser
```

Comment:
- [ ] Kindly in your README file write the steps that the user should follow to clone your repo ie.
- open your terminal and write `git clone https://github.com/LIBERCOSOFT/hello-microverse.git`.
- go to the `hello-microverse` directory.
- etc.